packages/@aws-cdk/aws-appmesh/package.json

```diff
@@ -108,10 +108,35 @@
       "construct-ctor-props-type:@aws-cdk/aws-appmesh.ImportedVirtualNode",
       "construct-ctor-props-type:@aws-cdk/aws-appmesh.ImportedVirtualRouter",
       "construct-ctor-props-type:@aws-cdk/aws-appmesh.ImportedVirtualService",
+      "from-signature:@aws-cdk/aws-appmesh.GatewayRoute.fromGatewayRouteName",
       "from-signature:@aws-cdk/aws-appmesh.Route.fromRouteName",
+      "from-signature:@aws-cdk/aws-appmesh.VirtualGateway.fromVirtualGatewayName",
```

Comment:
Since there's no such method anymore, pretty sure this line can be deleted 🙂

Reply:
I coulda swore I got rid of this. Must of been in the commit I reverted D: